client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/hooks/usePreviousNextItem.ts

```diff
@@ -0,0 +1,40 @@
+interface itemId {
+  id: string;
+}
+
+interface prevNextState {
+  hasPrevious: boolean;
+  previous: null | string;
+  hasNext: boolean;
+  next: null | string;
```

Comment:
in Manual reqs we have this as a structure with an id in it, I'm assuming we just want an ID here, but if that is the case we probably should name it `nextId` or something.
This could be a generic/common component if we only care about the id...

Reply:
Agree, should be a common component, doesn't just need to be for items either. Would just need to change the input "items" to be an array of strings rather than a specific type of object. Or, an additional prop to specify the field to compare against to avoid having to map the input array to just the IDs.